ckanext/opendata_theme/base/helpers.py

```diff
@@ -121,6 +121,14 @@ def is_data_dict_active(ddict):
     return False
 
 
+def is_segment_active():
+    return config.get('ckan.segment_active', 'false').lower() == 'true'
+
+
+def get_segment_apikey():
+    return str(config.get('ckan.segment_apikey', 'xBaaEbCBAwfUvAab29dAvDNL9GCH6Z4a'))
```

Comment:
Should this be called `writekey` instead?
https://segment.com/docs/connections/find-writekey/

Reply:
yeap